policyengine_us/variables/household/demographic/person/occupation.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_us.model_api import *
+
+
+class Occupation(Enum):
+    OTHER = "other"
+    MANAGEMENT_BUSINESS_FINANCIAL = "management, business and financial"
+    PROFESSIONAL_RELATED = "professional and related"
+    SERVICE = "service"
+    SALES = "sales and related"
+    OFFICE_ADMINISTRATIVE = "office and administrative support"
+    FARMING_FISHING_FORESTRY = "farming, fishing and forestry"
+    CONSTRUCTION_EXTRACTION = "construction and extraction"
+    INSTALLATION_MAINTENANCE_REPAIR = "installation, maintenance and repair"
+    PRODUCTION = "production"
+    TRANSPORTATION_MATERIAL_MOVING = "transportation and material moving"
+    MILITARY_SPECIFIC = "military"
+
+
+class occupation(Variable):
+    value_type = Enum
+    possible_values = Occupation
+    default_value = Occupation.OTHER
+    entity = Person
+    label = "occupation"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        occupation_code = person("occupation_code", period)
```

Comment:
let's call this major_occupation_recode for consistency with asec

ideally use OCCUP for last year's but if that's detailed and we'd need to map, let's file an issue to come back to it